test/specs/login.js

```diff
@@ -0,0 +1,25 @@
+const assert = require('assert');
+
+describe('ClickUp login', () => {
+  it('logs the user to clickUp with an invalid', () => {
```

Comment:
please use : 'should', 'shouldn't', something like this:
`it('shouldn't be able to log in with incorrect password')`